client/src/services/auth/authService.js

```diff
@@ -0,0 +1,18 @@
+import requestService from '@/services/requestService';
+
+const authService = {
+    async signIn(userLoginData) {
+        return await requestService.post('/auth/login', userLoginData);
+    },
+    async signUp(userRegisterData) {
+        return await requestService.post('/auth/register', userRegisterData);
+    },
+    async signOut() {
+        return await requestService.post('/auth/logout');
```

Comment:
when you signout, and got success response, remove token from localStorage by calling saveToken('')